<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PKUToDo1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/widgets_icons/check-sign-in-a-rounded-black-square.png</normaloff>:/icons/widgets_icons/check-sign-in-a-rounded-black-square.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_home">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/widgets_icons/house-outline.png</normaloff>:/icons/widgets_icons/house-outline.png</iconset>
       </attribute>
       <attribute name="title">
        <string>主页</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1" rowspan="2" colspan="5">
         <widget class="QLabel" name="label_name">
          <property name="font">
           <font>
            <family>Curlz MT</family>
            <pointsize>100</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PKU ToDo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>214</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>214</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="5">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>214</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="3">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>146</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>146</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>146</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>214</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_intro">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>项目介绍</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_calendar">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/widgets_icons/icons8-calendar-50.png</normaloff>:/icons/widgets_icons/icons8-calendar-50.png</iconset>
       </attribute>
       <attribute name="title">
        <string>日历</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_toDo">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/widgets_icons/checked.png</normaloff>:/icons/widgets_icons/checked.png</iconset>
       </attribute>
       <attribute name="title">
        <string>代办</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_search">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/widgets_icons/icons8-search-80.png</normaloff>:/icons/widgets_icons/icons8-search-80.png</iconset>
       </attribute>
       <attribute name="title">
        <string>搜索</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_statistics">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/widgets_icons/statistic-64.png</normaloff>:/icons/widgets_icons/statistic-64.png</iconset>
       </attribute>
       <attribute name="title">
        <string>数据统计</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/widgets_icons/icons8-setting-50.png</normaloff>:/icons/widgets_icons/icons8-setting-50.png</iconset>
       </attribute>
       <attribute name="title">
        <string>设置</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPKUToDo">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuPKUToDo"/>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
